rfc/$template.md

```diff
@@ -0,0 +1,18 @@
+# RFC: *topic of the proposal*
```

Comment:
Suggestion: Add YYYY-MM-DD to the file name template as well - makes it easier to find recent RFCs

Reply:
Great idea, time-sortability FTW